addons/mail/static/src/rtc/rtc_service.js

```diff
@@ -93,10 +93,10 @@ export class Rtc {
     constructor(env, services) {
         this.env = env;
         /** @type {import("@mail/core/store_service").Store} */
-        this.store = services["mail.store"];
+        this.discussStore = services["discuss.store"];
```

Comment:
Same comment: accessing `discuss` from outside its folder is not correct.

Reply:
When I see the diff, I think it would have been easier to move `rtc` before channel member.
Pretty much all features that depend on channel members should be moved before the main model.

Reply:
same comment: I'll move the whole rtc folder to discuss.